### ISSUE TITLE
Copy content from Blender Artists to README.md

### ISSUE THREAD

**Issue by seanr** [samytichadou/CatHide#7]
Someone posted a direct link to this in a chat with very little context, so I had to search to find out what this was since the documentation here is nonexistent. You can basically just copy everything from [the Blender Artists page](https://blenderartists.org/t/cathide-resurrected-for-2-8x-3d-view-category-hiding/692906) (converting the HTML to Markdown) and consider this done.

**Comment by samytichadou** [comment]
yup, i'll add a proper readme asap ! thanks for the suggestion !
cheers !
